addons/sale/wizard/sale_quotation_send_views.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="sale_quotation_send_views_form" model="ir.ui.view">
+        <field name="name">sale.quotation.send.form</field>
+        <field name="model">sale.quotation.send</field>
+        <field name="arch" type="xml">
+            <form string="Send By Email">
+                <field name="active_draft_ids" invisible="1"/>
+                <field name="active_sale_ids" invisible="1"/>
+                
+                <!-- If there is at least a draft, this is visible -->
```

Comment:
Good job on adding comments to your code!
Now, more often than not, the reader needs the rationale behind the code, rather than a description of what the code does. I believe that it's the case here since the comment is the "human" translation of the `attrs` below.
```suggestion
                <!-- If the user didn't select any draft order, hide the related template -->
```